docs/BreakingChanges/063 Apps published with ClickOnce that use a SHA-256 c.md

```diff
@@ -0,0 +1,33 @@
+## 63: Apps published with ClickOnce that use a SHA-256 code-signing certificate.
```

Comment:
{
  "Id": "63",
  "Title": "Apps published with ClickOnce that use a SHA-256 code-signing certificate.",
  "Details": "TODO",
  "Suggestion": "TODO",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#ClickOnce",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis?"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "Only affects WinServer 2003; there is a KB to address it.  Could flag it if we detect click-once being used (especially if it has a SHA-1 code-signing cert)\nSource analyzer status: Pri 3, Planned - No tracking work item yet",
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis?